src/app/recents/recents.component.html

```diff
@@ -0,0 +1,14 @@
+<app-spinner [loading]="loading"></app-spinner>
+<app-navbar></app-navbar>
+
+<div class="recent-title">{{ title }}</div>
+<div class="recent-tabs">
+  <tabset>
+      <tab customClass="recent-table" *ngFor="let type of types" heading="{{ type }}" id="{{ type }}">
```

Comment:
A more natural syntax here would be to use data-binding instead of interpolation as shown below. Although, they produce the same result, I believe an extra step is involved for Angular if you use interpolation.

`[heading]="type" [id]="type"`